source/module_hamilt_lcao/hamilt_lcaodft/FORCE_k.cpp

```diff
@@ -985,7 +989,7 @@ void Force_LCAO_k::cal_fvnl_dbeta_k(const elecstate::DensityMatrix<std::complex<
                             continue;
                         }
                         std::vector<double*> tmp_matrix_ptr;
-                        for (int is = 0; is < GlobalV::NSPIN; ++is)
```

Comment:
Here the code is changed due to error in calculation when get_DMR_pointer in nspin=4 case. It is not physical reason as the function should not be called in non-collinear calculation